dib/src/components/RestaurantCard/RestaurantCard.js

```diff
@@ -1,39 +1,57 @@
 /* eslint-disable jsx-a11y/img-redundant-alt */
-import "./RestaurantCard.css"
+import "./RestaurantCard.css";
 import { IoStar, IoEyeOff } from "react-icons/io5";
 import { Link } from "react-router-dom";
+import { useState } from "react";
 
 function RestCard({ rest }) {
-  const imageUrl = rest.photos && rest.photos.length > 0 && rest.photos[0].imageURL;
+    const imageUrl = rest.photos && rest.photos.length > 0 && rest.photos[0].imageURL;
 
-    // Shorten names longer than 25 characters
+    // shorten names longer than 25 characters
     let name = rest.name;
```

Comment:
I already created a similar logic just right up here, even adding "..." in the end, if the text is longer than x. i think it would be good to try to use the same way of writing it, when its in the same file.